tests/cephfs/cephfs_utilsV1.py

```diff
@@ -2916,3 +2915,13 @@ def generate_fio_config(ioengine, mount_dir, workload, size, iodepth, numjobs):
             fio_filenames.append(filename)
         log.info("Generated all the configs")
         return fio_filenames
+
+    @retry(CommandFailed, tries=3, delay=60)
```

Comment:
Should we increate the tries or delay for baremetal

Reply:
While testing, the services were up in 1st attempt only.
If we hit any issues later, we can increase the delays and retry count.